src/webgl/draw.js

```diff
@@ -23,12 +23,12 @@ export function draw(gl, {
 
   drawMode = glGet(drawMode);
   indexType = glGet(indexType);
-
-  assertDrawMode(drawMode, 'in draw');
-  if (isIndexed) {
-    assertIndexType(indexType, 'in draw');
+  if (gl.debug) {
```

Comment:
+1

Actually, I think we should just delete these, no need to keep them in an if statement. This type of checks just make the executable bigger and any errors will already be caught in debug mode by the `WebGLDebug` package.